bin/environmentd-debug-zip

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+#
+# materialized — build and run a core service or test.
+
+if [ -z "${MZDEV_POSTGRES}" ]; then
+    POSTGRES='postgres://root@localhost:26257/materialize'
+else
+    POSTGRES=${MZDEV_POSTGRES}
+fi
+
+echo "Shutting down running environmentd and clusterd (if any)"
+killall -q environmentd
+killall -q clusterd
+
+for i in consensus.consensus storage.collections storage.data storage.fence storage.sinces storage.uppers tsoracle.timestamp_oracle adapter.collections adapter.data adapter.fence adapter.sinces adapter.uppers; do
+  echo "Exporting ${i} from CockroachDB"
+  psql "${POSTGRES}" -q --csv -c "select * from ${i}" > mzdata/persist/${i}.csv;
+done
+
+zip -qr mzdata/persist.zip mzdata/persist
```

Comment:
I feel like you might want to call this `debug.zip` instead of `persist` to avoid the impression that this is just `persist` state? Also seems worthwhile to bundle up all of mzdata—doesn't hurt to get the secrets and the environment-id too.

```suggestion
zip -qr mzdata/debug.zip mzdata
```

Reply:
Done, except I didn't want to accidentally include a previous debug.zip if it exists (recursive debug.zip!), so explicitly listed `mzdata/persist mzdata/environment-id mzdata/secrets`. We'll have to maintain that list if anything gets added, but seems fine?